report/5.Solution/FromScreenToMotors.tex

```diff
@@ -1,85 +1,77 @@
 $$
 
 \subsection{From USB to motors}
-The NXT receives a vector in range (-127 -> 127, -127 -> 127).
-This ought to get mapped to the speed of the motor which is a value of -100 and 100, on each axis.
-Either full speed in the negatie direction or full speed in the positive direction.
-
-
-The output of the USB is an 8 bit int, and the input of the motor is also an int.
-The easiest way to do these calculations are simply mapping the ranges directly onto each other
+The NXT receives a vector in range $(-127 -> 127, -127 -> 127)$. \todo{correct mathematical notation?}
+This vector should be mapped to the speed of the motor which is a value between $-100$ and $100$ on each axis where the number corrosponds to a percentage-wise speed in either a positive or negative direction.
 
-However we never want full speed (as that is quite extreme) nor do we want 0 speed (as the motors oftentimes won't move before reaching a power of 10-20 due to sheer weight).
-Therefore the actual range is something like -30 and -10 and 10 and 30
+The output of the USB is an 8-bit integer, and the input of the motor is also an int.
+The easiest way to do these calculations is by simply mapping the ranges directly onto one another.
+However, we rarely want full speed, nor do we want 0 speed as the motors oftentimes will not move until reaching a power level of 10 to 20 due to sheer weight of the attached LEGO components.
+Therefore. the actual range was decided to be between $-30$ and $-10$ and likewise in the positive direction.
 
-These values of $\pm10$ and $\pm30$ should obviously be tweakable, but in these examples, 10 and 30 will be used for the sake of examples. 
+These values of $\pm10$ and $\pm30$ should obviously be tweakable, but in these examples, 10 and 30 will be used for the sake of simplicity. 
 Input, in these constants, is the input of the function, which is the output of the USB module.
 
-	lower\_bound = 10
-	upper\_bound = 30
-	input\_upperbound = 127
-	range = upper\_bound - lower\_bound
+	$$lower\_bound = 10$$
+	$$upper\_bound = 30$$
+	$$input\_upperbound = 127$$
+	$$range = upper\_bound - lower\_bound$$
 
-normally mapping from input to output would be making the input into a range of -1 and 1, however this required floating point precision (otherwise we simply get -1, 0 or 1) so instead we do it in a bit of an other order.
+Normally, mapping from input to output would be translating the input into a range of $-1$ to $1$, however, this required floating point precision since it would otherwise result in simply getting -1, 0 or 1, so instead the order is changed up a bit:
 
 $$
 speed(pos) = (pos \cdot range)/input_\text{upperbound} + (dir(pos) \cdot lowerbound)
 $$
 
-An issue with direction rises again with direction, bacause of lowerbound.
-This is fixed by using the direction, so we actually uses the bound correctly.
\ No newline at end of file
+An issue with direction rises again with direction, because of lowerbound.
```

Comment:
No clue. I think it has something to do with the lowebound -127 giving issues. Ask casper directly